projects/admin/src/app/record/detail-view/acq-order-detail-view/acq-order-detail-view.component.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { DetailRecord } from '@rero/ng-core/lib/record/detail/view/detail-record';
+import { Observable } from 'rxjs';
+import {RecordEvent, RecordService} from '@rero/ng-core';
+
+@Component({
+  selector: 'admin-order-detail-view',
+  templateUrl: './acq-order-detail-view.component.html'
+})
+export class AcqOrderDetailViewComponent implements OnInit, OnDestroy, DetailRecord {
+
+  /** Observable resolving record data */
+  record$: Observable<any>;
+
+  /** Resource type */
+  type: string;
+
+  /** AcqOrder record */
+  record: any;
+
+  /** The observer to the record observable */
+  private recordObs = null;
+
+  /** AcqOrder total amount */
+  totalAmount: number;
+
+  /**
+   * Constructor
+   */
+  constructor(
+    private recordService: RecordService
+  ) { }
+
+  /**
+   * On init hook
+   */
+  ngOnInit() {
+    this.recordObs = this.record$.subscribe((record: any) => {
+      this.record = record;
+      this.totalAmount = record.metadata.total_amount;
+    });
+  }
+
+  /**
+   * On destroy hook
+   */
+  ngOnDestroy() {
+    this.recordObs.unsubscribe();
+  }
+
+  /**
+   * Get order total amount
+   * @return number - total amount
+   */
+  get orderTotalAmount(): number {
+    return this.totalAmount;
+  }
+
+  /**
+   * Delete order line event
+   * @param orderPid - Order PID
+   */
+  deleteOrderLine(orderPid: string) {
+    this.recordService.getRecord(
```

Comment:
The method suggest that it will delete a order line but it update the total instead...

Reply:
changed